docker-app/qfieldcloud/core/utils2/projects.py

```diff
@@ -81,10 +81,8 @@ def create_collaborator_by_username_or_email(
             username
         )
     elif users[0].is_organization:
-        message = (
-            _(
-                'Organization "{}" cannot be added. Only users and teams can be collaborators.'
-            ).format(username),
+        message = _(
+            f'Organization "{username}" cannot be added. Only users and teams can be collaborators.'
```

Comment:
@faebebin @why-not-try-calmer 

I have already commented this in another place, this replacing `"{}".format(1)` with `f"{1}"` is causing troubles for potential translations.

Reply:
Fixed.